local/bin/sh/run-site.sh

```diff
@@ -14,6 +14,7 @@ if [ ${RUN_SERVER} == true ]; then
 		echo "checking that node modules are installed and up-to-date"
 		npm install
         echo "starting gulp watch"
+        gulp build
```

Comment:
we need to make sure we build every startup, which will create any missing manifests